securesignals-ima-dev-app/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.dev'
+
+    defaultConfig {
+        applicationId "com.uid2.securesignals.ima.devapp"
+        minSdk 21
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Nit: Not required since the Dev-App doesn't have any Android Tests.

Reply:
Done.